e2e/e2eutil/acl.go

```diff
@@ -0,0 +1,49 @@
+package e2eutil
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	api "github.com/hashicorp/nomad/api"
```

Comment:
```suggestion
	"github.com/hashicorp/nomad/api"
```

Reply:
What is wrong with my editor config that it did this?! :thinking: